### ISSUE TITLE
JIT/Regression/VS-ia64-JIT/V1.2-M02/b12011/ failing on *nix

### ISSUE THREAD

**Issue by pgavlin** [dotnet/coreclr#2300]
This test involves an asembly whose name in metadata is not the same as the name of the file on disk, and fails with the following log on *nix:

```
MESSAGE:
BEGIN EXECUTION /mnt/j/workspace/dotnet_coreclr/release_ubuntu_tst_prtest/bin/tests/Windows_NT.x64.Release/Tests/coreoverlay/corerun b12011.exe libust[3400/3400]: Warning: HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) Unhandled Exception: System.TypeLoadException: Could not load type 'ShellApplication' from assembly 'attrib014_2, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. ./b12011.sh: line 52: 3400 Aborted (core dumped) $_DebuggerFullPath "$CORE_ROOT/corerun" "b12011.exe" $CLRTestExecutionArguments $Host_Args Expected: 100 Actual: 134 END EXECUTION - FAILED
+++++++++++++++++++
STACK TRACE:
```

**Comment by sergiy-k** [comment]
This test fails on Unix because it requires the COM Interop feature which is enabled on Windows only. So the failure in this test is expected on Unix platforms.

**Comment by pgavlin** [comment]
Closed by #4831.
